x/mongo/driver/operation/find.go

```diff
@@ -173,6 +177,18 @@ func (f *Find) command(dst []byte, desc description.SelectedServer) ([]byte, err
 	return dst, nil
 }
 
+// AllowDiskUse when true allows temporary data to be written to disk during the find command. Valid for server
```

Comment:
This comment is slightly different than the one on FindOptions.  Seems like they should be the same.